CMakeLists.txt

```diff
@@ -25,7 +25,9 @@ if( GCC )
 endif()
 if(BUILD_WITH_CLANG)
   set(CMAKE_CXX_COMPILER "${BUILD_WITH_CLANG}/bin/clang++")
-  set(CMAKE_CC_COMPILER "${BUILD_WITH_CLANG}/bin/clang")
+  file(TO_CMAKE_PATH "${BUILD_WITH_CLANG}" CLANG_PATH)
```

Comment:
Why is this needed?  Why not use TO_NATIVE_PATH?

Reply:
TO_NATIVE_PATH is not working for me if I use 

-DBUILD_WITH_CLANG=~/llvm

If I expand ~/llvm to the full absolute path it works. 

I'm on Ubuntu 19.04 and not sure if this will work on Windows though.